commit f5a782337870ed3edca223bfee8645b806a8c7e7
Author: Burkhard Lück <lueck@hube-lueck.de>
Date:   Sun May 11 16:39:00 2014 +0200

    add missing X-DocPath + fix wrong HTML_INSTALL_DIR for network kioslave

diff --git a/doc/kioslave/network/CMakeLists.txt b/doc/kioslave/network/CMakeLists.txt
index dd75aec..707228e 100644
--- a/doc/kioslave/network/CMakeLists.txt
+++ b/doc/kioslave/network/CMakeLists.txt
@@ -1 +1 @@
-kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/en)
+kde4_create_handbook(index.docbook INSTALL_DESTINATION ${HTML_INSTALL_DIR}/en SUBDIR kioslave/network)
diff --git a/kioslave/network/ioslave/network.protocol b/kioslave/network/ioslave/network.protocol
index d735646..7bec683 100644
--- a/kioslave/network/ioslave/network.protocol
+++ b/kioslave/network/ioslave/network.protocol
@@ -4,6 +4,8 @@ protocol=network
 input=none
 output=filesystem
 
+X-DocPath=kioslave/network/index.html
+
 Icon=network-workgroup
 Description=A kioslave to browse the network
 Description[ar]=خادم دخل وخرج كدي لتصفح الشبكة
